applications/nrf_desktop/README.rst

```diff
@@ -8,15 +8,25 @@ Nordic Semiconductor HID reference design.
 Overview
 ********
 
-This application is a reference design of a HID device.
-
 The code is currently work-in-progress and as such is not fully functional, verified, or
```

Comment:
I'd move this sentence to just before  Requirements, and highlight it with a note.

Reply:
done